src/send/SendEnterAmount.tsx

```diff
@@ -383,6 +392,15 @@ function SendEnterAmount({ route }: Props) {
             testID="SendEnterAmount/NotEnoughForGasWarning"
           />
         )}
+        {prepareTransactionError && (
+          <InLineNotification
+            severity={Severity.Error}
+            title={t('sendEnterAmountScreen.feeEstimationError.title')}
+            description={t('sendEnterAmountScreen.feeEstimationError.description')}
+            style={styles.warning}
+            testID="SendEnterAmount/PrepareTransactionError"
+          />
+        )}
```

Comment:
Nothing to do now, but it could be a good idea to surface some details about the error (like we recently did for WC prepare transactions failures in https://github.com/valora-inc/wallet/pull/4864).

Reply:
oh, interesting attack vector 🤔